src/main/blackbox/blackbox.c

```diff
@@ -546,6 +553,9 @@ static void writeIntraframe(void)
      */
     blackboxWriteUnsignedVB(blackboxCurrent->rcCommand[THROTTLE] - motorConfig()->minthrottle);
 
+    // Write setpoint roll, pitch, yaw and unused 4th value (always 0):
```

Comment:
I think this comment does not apply in the latest version (if I'm not wrong in the 4th value it has the throttle)